pages/components/text.html

```diff
@@ -105,8 +105,31 @@ <h1 clss="page-title">text</h1>
 
 
   <main class="main-content">
-
-    <iframe src="/codeblock/x.txt" frameborder="0" class="codeblock"></iframe>
+    <div>
```

Comment:
just a suggestion, nice work adding main semantic tag, you can add more semantic tags like section instead of the divs below